awesomplete.js

```diff
@@ -301,9 +309,7 @@ function $$(expr, con) {
 	return slice.call((con || document).querySelectorAll(expr));
 }
 
-$.create = function(tag, o) {
-	var element = document.createElement(tag);
-
+$.set = function(element, o) {
```

Comment:
I guess it depends on how much of Awesomplete is in Bliss now :) If you think all of Bliss code is not too big to be used for Awesomplete, then it's fine to use it.

If Awesomplete and Bliss will use ES6 modules one day, then the answer would be definitely yes. With ES6 modules it would be so much easier to include only the necessary parts of Bliss into Awesomplete build. Of course this can be done without ES6 modules, but it's kind of a hack.